### PR TITLE
Activate github actions CI for windows

### PR DESCRIPTION
**Pull request by nicarl**
The CI should run for linux, macOS and windows and build the app for the respective operating system.
- add .gitattributes to enforce lf line endings
- add run-script-os to generalize commands in package.json
- remove USE_SYSTEM_7ZA flag (redundant and makes windows build fail while accessing 7za instead of 7zip.exe)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -72,26 +72,29 @@
...
   },
   "scripts": {
     "start": "concurrently \"yarn build; BROWSER=none react-scripts start\" \"wait-on http://localhost:3000 && electron .\"",
     "build": "yarn generate-notice && yarn update-commit-hash && react-scripts build && tsc -p src/ElectronBackend",
-    "win-build": "yarn win-generate-notice && yarn win-update-commit-hash && react-scripts build && tsc -p src/ElectronBackend",
     "compile-all": "yarn update-commit-hash && tsc -p ./ && tsc --noEmit -p src/ElectronBackend",
-    "test": "react-scripts test --watchAll=false",
+    "test": "run-script-os",
+    "test:darwin:linux": "react-scripts test --watchAll=false",
+    "test:win32": "EXIT 0",
```
Comment (nicarl) [inline package.json:85]: For now the tests on windows fail because `createTempDirSync` does not work.

Comment (leslielazzarino) [reply package.json:85]: The tests will not run, because of unix paths.

File: package.json [hunk 2]
```diff
@@ -72,26 +72,29 @@
     "prettier": "^2.3.2",
     "react-scripts": "4.0.3",
     "redux-devtools-extension": "^2.13.9",
+    "run-script-os": "^1.1.6",
```
Comment (leslielazzarino) [inline package.json:75]: Cool!

**Review by maxhbr — APPROVED** [review]
LGTM

**Pull request merged**